client/src/components/about/index.js

```diff
@@ -1,8 +1,60 @@
 import React from 'react'
+import { RaisedButton, Paper } from 'material-ui'
+
+import './style.css'
+
+const githubIcon = require('../../assets/icons/GitHub-Mark-32px.png')
+const MESAIcon = require('../../assets/images/MESA_logo.svg')
 
 export default () => (
-  <div>
-    <h1>About Us</h1>
-    <p>Hello Medium!</p>
+  <div className='about-main-div'>
+    <div className='about-inner-div'>
+      <Paper className='about-paper' zDepth={2}>
+        <h1 className='about-header-one'>About MESA</h1>
+
+        <h3 className='about-header-three'>Mission</h3>
+        <p className='about-mission'>
+          To provide students underrepresented in the fields of mathematics, engineering, science, and technology with the skills, knowledge, and opportunities to develop their talents, explore technology-based careers, enter college, and compete successfully in the workforce.
+        </p>
+
+        <div className="about-mesa-button-wrapper">
+          <RaisedButton backgroundColor='#ff9e15' label='Learn more about MESA' fullWidth={true} />
+        </div>
+        
+      </Paper>
+      <Paper className='about-paper'>
+        <h2 className='about-header-one'>About This App</h2>
+        <p className='about-text'>
+          MESA Roadmaps lets users plot out their career, including education and work experiences, as well as clubs, activities, and other programs they were involved in at these institutions. The user's career is then laid out on a Google Maps interface. The app is designed for students interested in STEM to see just how professionals in the field got to where they are and to see all the different routes that lead to success.
+        </p>
+
+      </Paper>
+      <Paper className='about-paper'>
+        <h2 className='about-header-one'>About Us</h2>
+        <p className='about-text'>
+          Created by 
+        </p>
+        <p className='about-text'>
+          <p>Ian Dusenbury</p>
```

Comment:
Getting a console error here saying p tags shouldn't be nested with other p tags.